pkg/utils/encrypt.go

```diff
@@ -43,3 +43,17 @@ func Decrypt(ciphertextB64 string) (string, error) {
 	}
 	return string(plaintext), nil
 }
+
+// DecryptOrPlaintext tries to decrypt base64 encoded ciphertext to plaintext or return plaintext
+func DecryptOrPlaintext(ciphertextB64 string) (string, error) {
+	ciphertext, err := base64.StdEncoding.DecodeString(ciphertextB64)
+	if err != nil {
+		return ciphertextB64, nil
+	}
+
+	plaintext, err := encrypt.Decrypt(ciphertext)
+	if err != nil {
+		return "", terror.Annotatef(err, "can not decrypt password %s", ciphertextB64)
+	}
+	return string(plaintext), nil
```

Comment:
Maybe I should return the plaintext and error when it goes wrong, and then log warnning externally